code/modules/mob/living/simple_animal/hostile/ordeal/misc.dm

```diff
@@ -26,6 +26,10 @@
 	//Funny drags everything to it
 /mob/living/simple_animal/hostile/ordeal/pink_midnight/proc/Breach_All()
 	for(var/mob/living/simple_animal/hostile/abnormality/A)
```

Comment:
I just realized what the hell you are doing and why it is lagging. You are iterating through EVERY OBJECT IN THE WHOLE GAME WORLD.
Kirie...

Reply:
Jesus fuckign christ, I thought it defaulted to "In mob global list"